// @shopify/polaris/build/cjs/components/Badge/types.js
'use strict';

exports.ToneValue = void 0;
(function (ToneValue) {
  ToneValue["Info"] = "info";
  ToneValue["Success"] = "success";
  ToneValue["Warning"] = "warning";
  ToneValue["Critical"] = "critical";
  ToneValue["Attention"] = "attention";
  ToneValue["New"] = "new";
  ToneValue["InfoStrong"] = "info-strong-experimental";
  ToneValue["SuccessStrong"] = "success-strong-experimental";
  ToneValue["WarningStrong"] = "warning-strong-experimental";
  ToneValue["CriticalStrong"] = "critical-strong-experimental";
  ToneValue["AttentionStrong"] = "attention-strong-experimental";
  ToneValue["ReadOnly"] = "read-only-experimental";
  ToneValue["Enabled"] = "enabled-experimental";
})(exports.ToneValue || (exports.ToneValue = {}));
exports.ProgressValue = void 0;
(function (ProgressValue) {
  ProgressValue["Incomplete"] = "incomplete";
  ProgressValue["PartiallyComplete"] = "partiallyComplete";
  ProgressValue["Complete"] = "complete";
})(exports.ProgressValue || (exports.ProgressValue = {}));
